packages/graphql/src/types.ts

```diff
@@ -335,3 +332,13 @@ export interface NestedRecord<T> extends Record<string | symbol | number, T | Ne
 
 /** Input field for graphql-compose */
 export type InputField = { type: string; defaultValue?: string } | string;
+
+export interface Neo4jGraphQLJWTPlugin {
```

Comment:
**suggestion:** would the suggestion below not be the accurate name here?

```suggestion
export interface Neo4jGraphQLAuthPlugin {
```

Reply:
Sure